### PR TITLE
[ONNX] load model API

### PR DESCRIPTION
**Pull request by cynthia-liu**

### CODE REVIEW

**Comment by cynthia-liu** [comment]
ZOO-PR-Validation - Build # 2048 - Successful:

Check console output at http://10.239.47.211:18888/job/ZOO-PR-Validation/2048/ to view the results.

The pull request: https://github.com/intel-analytics/analytics-zoo/pull/1237 Successful

**Comment by cynthia-liu** [comment]
ZOO-PR-Validation - Build # 2061 - Successful:

Check console output at http://10.239.47.211:18888/job/ZOO-PR-Validation/2061/ to view the results.

The pull request: https://github.com/intel-analytics/analytics-zoo/pull/1237 Successful

File: pyzoo/test/zoo/pipeline/onnx/test_model_loading.py [hunk 1]
```diff
@@ -86,6 +86,7 @@ def test_onnx_batch_norm1(self):
         pytorch_model = torch.nn.Sequential(
             torch.nn.BatchNorm2d(num_features=3, momentum=1, affine=False)
         )
+        pytorch_model.eval()
```
Comment (zhichao-li) [inline pyzoo/test/zoo/pipeline/onnx/test_model_loading.py]: remove this and add the changes into `test_utils_onnx.py`?

File: pyzoo/test/zoo/pipeline/onnx/test_model_loading.py [hunk 2]
```diff
@@ -94,6 +95,7 @@ def test_onnx_batch_norm2(self):
         pytorch_model = torch.nn.Sequential(
             torch.nn.BatchNorm2d(num_features=3, momentum=1, affine=True)
         )
+        pytorch_model.eval()
```
Comment (zhichao-li) [inline pyzoo/test/zoo/pipeline/onnx/test_model_loading.py]: remove.

File: pyzoo/test/zoo/pipeline/utils/test_utils_onnx.py [hunk 3]
```diff
@@ -75,7 +74,7 @@ def compare_with_pytorch(self, pytorch_model, input_shape_with_batch,
         # coutput = caffe2.python.onnx.backend.run_model(onnx_model, input_data_with_batch)[0]
 
         pytorch_out = pytorch_model.forward(self._convert_ndarray_to_tensor(input_data_with_batch))
```
Comment (zhichao-li) [inline pyzoo/test/zoo/pipeline/utils/test_utils_onnx.py]: add  ` pytorch_model.eval()` before `forward`

**Comment by cynthia-liu** [comment]
ZOO-PR-Validation - Build # 2066 - Successful:

Check console output at http://10.239.47.211:18888/job/ZOO-PR-Validation/2066/ to view the results.

The pull request: https://github.com/intel-analytics/analytics-zoo/pull/1237 Successful

**Pull request merged**